pkgs/os-specific/linux/tuxedo-keyboard/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, kernel, kmod }:
+
+# TODO: look at the other kernel modules packages and see if you find improvements to do
```

Comment:
That part at `installPhase` seems still a bit off. Creating a directory just to move some artefact away seems redundant. Maybe I could find some way to direclty output the module to `$out/lib/modules/${kernel.modDirVersion}`.